test/acceptance/tests/basic/basic_test.go

```diff
@@ -1031,6 +1061,21 @@ func TestBasic(t *testing.T) {
 	}
 }
 
+// discoverConsulEnterpriseImage looks in mesh-task for the default Consul image
+// and uses that that same version of the enterprise image.
+func discoverConsulEnterpriseImage(t *testing.T) string {
+	filepath := "../../../../modules/mesh-task/variables.tf"
+	data, err := os.ReadFile(filepath)
+	require.NoError(t, err)
+
+	// Find the default consul image in the mesh-task module and use that same version.
+	re := regexp.MustCompile(`default\s+=\s+"public.ecr.aws/hashicorp/consul:(.*)"`)
+	matches := re.FindSubmatch(data)
+	require.Len(t, matches, 2) // entire match + one submatch
+	version := string(matches[1]) + "-ent"
+	return "public.ecr.aws/hashicorp/consul-enterprise:" + version
+}
```

Comment:
Thoughts on this? We could hardcode the enterprise image version to avoid this. We'd just need to remember bump that image version as well.